zeebe/gateway-grpc/src/main/java/io/camunda/zeebe/gateway/Gateway.java

```diff
@@ -391,6 +392,7 @@ private ServerServiceDefinition applyInterceptors(final BindableService service)
       } else {
         interceptors.add(new IdentityInterceptor(identityCfg, gatewayCfg.getMultiTenancy()));
       }
+      interceptors.add(new AuthenticationInterceptor());
```

Comment:
❓ We still support `AuthMode.NONE`. Do you think it makes sense to only add this interceptor if the `AuthMode` is `Identity`?

This way, the `AuthenticationInterceptor` will always be invoked, even when we want authentication to be disabled. And I think we might want this mode for local test setups.

Reply:
Yes, so the `AuthenticationInterceptor` shouldn't be invoked at all in that case.

Reply:
👍 Yes it is, and I mixed up the if-blocks in the code there. Sorry for wasting your time 😅